mlflow/models/utils.py

```diff
@@ -8,6 +8,7 @@
 
 import numpy as np
 import pandas as pd
+from pyspark.sql import DataFrame as SparkDataFrame
```

Comment:
If PySpark isn't installed, this import will fail. We should try/catch the import. Something like:

```
try:
    from pyspark.sql import DataFrame as SparkDataFrame

    PyFuncInput=... # Include Spark DataFrame
    PyFuncOutput=...
except Exception:
   PyFuncInput=... # Don't include Spark DataFrame
   PyFuncOutput=...
    pass

Reply:
How long does it take to import pyspark? If it's quick, it's fine to attempt to import it.